cli/tsc_config.rs

```diff
@@ -176,18 +214,13 @@ pub fn parse_config(
   assert!(!config_text.is_empty());
   let jsonc = jsonc_parser::parse_to_value(config_text)?.unwrap();
   let config: TSConfigJson = serde_json::from_value(jsonc_to_serde(jsonc))?;
-  let mut compiler_options: HashMap<String, Value> = HashMap::new();
-  let mut items: Vec<String> = Vec::new();
 
-  if let Some(in_compiler_options) = config.compiler_options {
-    for (key, value) in in_compiler_options.iter() {
-      if IGNORED_COMPILER_OPTIONS.contains(&key.as_str()) {
-        items.push(key.to_owned());
-      } else {
-        compiler_options.insert(key.to_owned(), value.to_owned());
-      }
-    }
-  }
+  let (compiler_options, items) =
```

Comment:
Yeah, this seems to be making things more complex then they need to be, IMO.  We already have too much indirection in the code, I would hate to continue to add back more.